mw-webapp/src/utils/auth/handleAuthState.ts

```diff
@@ -0,0 +1,12 @@
+import {User, onAuthStateChanged} from "firebase/auth";
+import {auth} from "src/firebase";
+
+export const handleUserAuthState = (setUserFunction: React.Dispatch<React.SetStateAction<User | null>>) => {
```

Comment:
Check naming. Function should have the same name as file

Reply:
Done